vnext/local-cli/src/config/projectConfig.ts

```diff
@@ -116,7 +116,7 @@ export function projectConfigWindows(
         projectFile:
           'Error: Project is required but not specified in react-native.config.',
       };
-    } else if (userConfig.project === null) {
+    } else if (!userConfig.project) {
```

Comment:
`=== null` means we didn't catch cases where it is not defined, since `null !== undefined`